### DISCUSSION TITLE
New 4th landcover class: irrigated crops (possible?)

### DISCUSSION THREAD

**Discussion by cuneyd** [mhm-ufz/mHM#53] [Q&A]
Hi all,

We have an expanding irrigation in the middle of a semi-arid region (Konya closed basin), where water supplied by groundwater extraction.

Choosing "pervious class" for all of the irrigated regions is not enough.

The "irrigated crops" cells should never have water limited situation due to irrigation mainly from GW.

Can I introduce a new LC class and write directly as fRoots0(k) = 1 in the mo_mpr_smhorizons.f90 file?
 
I assume that LAI correction just affects PET correction and not transpiration.

From the user forums I remember that mHM may not have irrigation scheme and reservoir operations yet.

But I hope we have a work around.. 

Cheers,
Cüneyd

**Comment by StephanThober** [comment]
Hi Cüneyd,

I think it would be great to have irrigation options in mHM. We should use the next user forum on April 6th to discuss this further. Will you be able to attend and even make a small presentation? What do you think?

Best,
Stephan

**Comment by cuneyd** [comment]
Sounds good,

See you on 6th of April.
